docs/en/device/npu.md

```diff
@@ -2,7 +2,8 @@
 
 ## Usage
 
-Please refer to [link](https://github.com/open-mmlab/mmcv/blob/master/docs/zh_cn/get_started/build.md) installing mmcv on NPU Devices.
+Please refer to [link](https://github.com/open-mmlab/mmcv/blob/master/docs/zh_cn/get_started/build.md) installing mmcv
```

Comment:
Please refer to the [building documentation of MMCV]() to install MMCV on NPU devices